### PR TITLE
Updated build documentation

### PR DESCRIPTION
**Pull request by DylanHBL**
Following an internal review, I've made changes to the build docs,

### CODE REVIEW

File: source/build/ukfast.md [hunk 1]
```diff
@@ -64,7 +66,6 @@ Below are the different facets that make up a UKFast solution, click on the link
       <h2>Hosted Desktop</h2>
       <ul>
           <li><a href="remotedesktop.html">Remote Desktop Services</a></li>
-          <li><a href="citrix.html">Citrix</a></li>
```
Comment (miff2000) [inline source/build/ukfast.md:67]: How come you're removing `Citrix`?

Comment (DylanHBL) [reply source/build/ukfast.md:67]: As we'd rather offer via FastDesk instead of dedi Citrix, Ant has requested it be removed.

**Comment by miff2000** [comment]
Is it worth removing the file from source control too, do you think?

**Comment by DylanHBL** [comment]
It can be but I wanted to keep it just in case someone wants it adding back in the future, I can just re-add the menu link.

**Review by miff2000 — APPROVED** [review]
LGTM

**Pull request merged**